composer.json

```diff
@@ -23,7 +23,8 @@
     },
     "require": {
         "akeneo/pim-community-dev": "^4.0.0",
-        "guzzlehttp/guzzle": "^7.2"
+        "guzzlehttp/guzzle": "^7.2",
+        "php-ds/php-ds": "^1.3"
```

Comment:
This uses the "soft" versions? Should we consider using the PECL extension?

https://github.com/iFixit/server-templates/blob/25a2f4caeb00388cd5da8be0ba4872e0d85e67aa/shared/scripts/install/php-ds#L3-L4

Reply:
Totally open to that, I went with the low-friction approach, but I'm not stuck in it. I'll try the pecl extension.

Reply:
Fixed!